coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift

```diff
@@ -82,7 +82,7 @@ struct DetailsScreen: View {
             
             // show popup on the whole screen
             if $showDeleteConfirmationPopup.wrappedValue {
-                DeleteConfirmationPopupView(showDeleteConfirmationPopup: $showDeleteConfirmationPopup)
+                DeleteConfirmationPopupView(showDeleteConfirmationPopup: $showDeleteConfirmationPopup, task: task!)
```

Comment:
```suggestion
                DeleteConfirmationPopupView(showDeleteConfirmationPopup: $showDeleteConfirmationPopup, task: task!)
```

Is there a way that we can pass a task and not have to force-unwrap it. 

If there is not, can we at least use a guard and an error instead. I would hate to have our app crash without some kind of message.

Reply:
@clc80, I don't think there is a way to pass task without it being optional. This is because while adding a new task, the task model won't be passed, and this can cause crashes in the onAppear closure of the view.